src/Relation/Nullary.agda

```diff
@@ -23,8 +23,9 @@ private
 ------------------------------------------------------------------------
 -- Re-exports
 
+open import Relation.Nullary.Recomputable public using (Recomputable)
 open import Relation.Nullary.Negation.Core public
-open import Relation.Nullary.Reflects public
+open import Relation.Nullary.Reflects public hiding (recompute)
```

Comment:
Unless I'm missing something, this is a breaking change and therefore we probably shouldn't do it in v2.1?

Reply:
Don't think so. 
Previously, `recompute` was defined in `Decidable.Core`, and not at all in `Reflects`.
Now it's defined in `Reflects` for `Reflects A b` instances, then used to re-define `recompute` in `Decidable.Core`, so it's simulatantouesly been newly introduced in `Reflects` and then hidden from clashing with the 'old' definition in `Decidable.Core` in this PR. So I think that's *not* a breaking change? Maybe the `CHANGELOG` needs fixing, but I think it's OK...?

Reply:
Do let me know if the present `CHANGELOG` correctly captures the refactoring (or not!)